benchmarks/spectre_v1/CMakeLists.txt

```
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 3.5.1)
+set(CMAKE_BUILD_TYPE Release)
+
+project(safeside VERSION 0.1.0 LANGUAGES CXX)
+
+# Use C++11 without extensions
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_EXTENSIONS off)
+
+add_executable(unmitigated microbenchmark.cc)
+target_compile_options(unmitigated PUBLIC -O0)
+
+if (CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
```

```suggestion
if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
```

for annoying reasons described in https://github.com/google/safeside/commit/9ca72f2dc0dbe8bb182bca534dd417c6fb6e558e